tests/init-global/warn/ScalaCheck.scala

```diff
@@ -0,0 +1,22 @@
+trait CmdLineParser:
+  outer =>
+
+  val a: String
+
+  trait Opt[+T]:
+    val default: T
+    val names: Set[String]
+    val help: String
+
+  trait IntOpt extends Opt[Int]:
+    println("outer = " + outer)
+    println("outer.a = " + outer.a)
+
+object FirstParser extends CmdLineParser:
+  object OptMinSuccess extends IntOpt:
+    val default = 100
+    val names = Set("bla")
+    val help = "bla"
+
+  val opts = List(OptMinSuccess)    // warn
```

Comment:
I'm getting `Warnings found on incorrect row numbers when compiling tests/init-global/warn/ScalaCheck.scala` when I run the test. The warning is on line 16, then has a trace that ends on 21.